hazelcast/src/main/java/com/hazelcast/cp/internal/RaftService.java

```diff
@@ -148,19 +158,19 @@
     private static final int AWAIT_DISCOVERY_STEP_MILLIS = 10;
 
     private final ReadWriteLock nodeLock = new ReentrantReadWriteLock();
-    @Probe
+    @Probe(name = CPSUBSYSTEM_METRIC_RAFT_SERVICE_NODES, unit = COUNT)
```

Comment:
Don't we use `COUNT` by default? Same question for other count probes here.

Reply:
Removed the unit.